lms/djangoapps/certificates/views/webview.py

```diff
@@ -53,10 +53,11 @@ def get_certificate_description(mode, certificate_type, platform_name):
     """
     certificate_type_description = None
     if mode == 'honor':
-        # Translators:  This text describes the 'Honor' course certificate type.
-        certificate_type_description = _("An {cert_type} Certificate signifies that an {platform_name} "
-                                         "learner has agreed to abide by {platform_name}'s honor code and "
-                                         "completed all of the required tasks for this course under its "
+        # Translators:  This text describes the 'Honor' course
+        # certificate type.
```

Comment:
@pdesjardins is it not possible to put all of this translators' comment on a single line?